packages/lab/src/parent-child-layout/ParentChildLayout.tsx

```diff
@@ -25,15 +17,11 @@ export interface ParentChildLayoutProps extends FlexLayoutProps<"div"> {
   /**
    * Change element that is displayed when in staked view.
    */
-  collapsedViewElement?: StackedViewElement;
+  collapseChildElement?: boolean;
   /**
    * Disable all animations.
    */
   disableAnimations?: boolean;
```

Comment:
should we be disabling animations on a component basis - feel like this should be something configured at the SaltProvider level ??